### PR TITLE
Convert Travis to Docker and autodeploy stage

### PR DESCRIPTION
**Pull request by pmac**

### CODE REVIEW

**Comment by pmac** [comment]
@willkg this should be pretty much just like the webapp. I removed docker-compose because it's not necessary since we only need the one container. Seems like the Makefile is a good enough place to put all of the commands and stuff. We can discuss further here though.

**Comment by willkg** [comment]
This is fine by me.

Does it help to wait on this until after we've merged standups and rogerroger?

File: Makefile [hunk 1]
```diff
@@ -16,24 +14,29 @@ help:
...
 
 build:
-	-rm .docker-build
-	${DOCKERCOMPOSE} -f docker-compose.yml build irc
+	docker build -t standup_irc .
 	touch .docker-build
 
 run: .docker-build
-	${DOCKERCOMPOSE} up irc
+	docker run --env-file .env_dev -v "$PWD:/app" standup_irc
```
Comment (willkg) [inline Makefile]: This doesn't account for the postgres requirement. Though maybe we don't need a run rule at all?

Comment (pmac) [inline Makefile]: hmm... there wasn't a postgres service setup in the docker-compose.yml file. I don't really think it's very useful locally, though I could see an argument for having that option. I think the bot does just fine w/o a DB if there isn't one, though testing the DB stuff is important. This is a good question. Perhaps we should keep the compose file even if we don't use it in the Makefile for when one might want to do a full-stack run locally. Or is it possible to switch over to SQLite for local?

Comment (willkg) [inline Makefile]: I read through the code and the pg stuff is optional. If it's there, then standup-irc will persist things like joined channels. If not, no big deal.

That covers my thoughts.

**Comment by willkg** [comment]
This looks good to me. r+

I'll let you merge it if/when you're comfortable.

**Pull request merged**